packages/vc-verification/src/issuer-verification.ts

```diff
@@ -90,10 +90,10 @@ export class IssuerVerification {
       if (!issuerCredential) {
         return verificationResult(false, ERRORS.NoCredential);
       }
-      if (
-        !(await this.revocationVerification.checkRevocationStatus(issuer, role))
-      ) {
-        return verificationResult(false, ERRORS.IssuerCredentialRevoked);
+      const revocationStatusResult =
+        await this.revocationVerification.checkRevocationStatus(issuer, role);
+      if (!revocationStatusResult.status) {
```

Comment:
I find this line pretty confusing because I typically expect `status` to be an enum of some kind instead of a boolean (e.g. HTTP status code are are large list)

This is probably a different task, but what do you think of either:
- renaming `status` to `verified` and keeping it as a boolean
- changing `status` to an enum of possible values (e.g. `VERIFIED`, `NOT_VERIFIED`)
- or some other option

Reply:
Hmm, renaming it to `verified` sounds good. IMO `boolean` fits the return type and eases the validation upon it.